moskito-backend/src/Serializer/CampsiteSerializer.php

```diff
@@ -2,22 +2,56 @@
 
 namespace App\Serializer;
 
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
 use App\Entity\Campsite;
+use App\Entity\CampsiteFeature;
+
 
 class CampsiteSerializer {
 
     private $elementAsArray = [];
 
+    private $featureNames = [
+        'wlan' => CampsiteFeature::TYPE_WLAN,  
```

Comment:
siehe Kommentar oben ... ;)
würde sich dann hier nämlich erledigen

Reply:
Aber an welcher Stelle übergebe ich dann den Konstanten-Namen? Vom Frontend bekomme ich direkt den Wert wlan, der in den setType('wlan') geht. Eigentlich soll hier die Konstante rein, aber die wird jetzt irgendwie obsolet.